### PR TITLE
Reduce local non-docker dependencies for update tests

### PR DESCRIPTION
**Pull request by cevian**
Get rid of dependencies on a local psql and stop mounting data volumes.

### CODE REVIEW

File: scripts/test_updates.sh [hunk 1]
```diff
@@ -30,39 +33,43 @@ docker rm -f timescaledb-orig timescaledb-updated timescaledb-clean || true
 rm -rf  ${CLEAN_DATA_DIR} ${UPDATED_DATA_DIR}
```
Comment (olofr) [inline scripts/test_updates.sh]: CLEAN_DATA_DIR and UPDATED_DATA_DIR is not used any more right ? One other place in the file as well.

Comment (cevian) [inline scripts/test_updates.sh]: Fixed

Comment (olofr) [inline scripts/test_updates.sh]: I still see them, did you forget to push '?

Comment (cevian) [inline scripts/test_updates.sh]: Yes. Really fixed now.

Comment (RobAtticus) [inline scripts/test_updates.sh]: Just to make sure, these are not macros used in the base PG Makefile, right?

Comment (cevian) [inline scripts/test_updates.sh]: Nope

Comment (cevian) [inline scripts/test_updates.sh]: That is they are not used, correct.

**Comment by cevian** [comment]
@erimatnor the last `docker rm -vf timescaledb-updated timescaledb-clean` command should remove the volume because of the `-v` no?

**Comment by cevian** [comment]
@erimatnor ping

**Pull request merged**